lib/jamaica/games/wordgrid_game.dart

```diff
@@ -0,0 +1,597 @@
+import 'dart:async';
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:maui/components/Shaker.dart';
+import 'package:maui/components/responsive_grid_view.dart';
+import 'package:maui/jamaica/state/game_utils.dart';
+
+class WordGridGame extends StatefulWidget {
```

Comment:
file name should be word_grid_game.dart...please pay attention to camelCase and snake_case

Reply:
changes done